packages/ketcher-react/src/typings.d.ts

```diff
@@ -10,6 +8,17 @@ declare module '*.sdf' {
   export default content
 }
 
+declare module '*.svg' {
+  import * as React from 'react'
```

Comment:
Import React inside the module looks a little bit weird to me. I checked another example in google and saw that it might be. What do you think? Could we move this import to the top of the file?